tests/unit/dataactvalidator/test_c9_award_financial.py

```diff
@@ -120,11 +120,21 @@ def test_zero_federal_action_obligation_and_positive_original_loan_subsidy_cost(
 
     tas = _TAS
     afa = AwardFinancialAssistanceFactory(tas=tas, fain='abc', uri=None, federal_action_obligation=0,
-                                          original_loan_subsidy_cost='1')
+                                          original_loan_subsidy_cost='1', assistance_type='08')
+    afa_2 = AwardFinancialAssistanceFactory(tas=tas, fain='abc', uri=None, federal_action_obligation=2,
```

Comment:
Function name makes it seem that it is only testing federal action obl with a value of 0, but here fed action obl has a value of 2.